### ISSUE TITLE
Macrome is unpublished

### ISSUE THREAD

**Issue by AmitDigga** [macrome-js/macrome#8]
`npx macrome watch`

```
npm ERR! code E404
npm ERR! 404 Not Found - GET https://registry.npmjs.org/macrome - Not found
npm ERR! 404
npm ERR! 404  'macrome@latest' is not in the npm registry.
npm ERR! 404 You should bug the author to publish it (or use the name yourself!)
npm ERR! 404
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/amit/.npm/_logs/2021-01-25T05_11_28_879Z-debug.log
Install for [ 'macrome@latest' ] failed with code 1
```

Am I missing something?

**Comment by conartist6** [comment]
Yeah, I haven't published it yet. It should be published in the next two weeks or so though since I've just shifted my full-time focus back to this project as of today.

I took a break from it to focus on shipping [iter-tools@7](https://www.npmjs.com/package/iter-tools/v/7.0.2) and [@iter-tools/regex](https://github.com/iter-tools/regex) (which I need in this project to efficiently parse headers). `iter-tools@7` is out and regex is pretty much ready for 1.0 so I'm currently testing it out with the macrome codebase, after which I'll attempt to build iter-tools with macrome and if all goes well I'll publish macrome.

**Comment by AmitDigga** [comment]
Oh, I didn't know that.  Good to know that the timeline for this is setup.
